specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json

```diff
@@ -2265,6 +2265,13 @@
         "fallbackRoute": {
           "description": "The properties of the route that is used as a fall-back route when none of the conditions specified in the 'routes' section are met. This is an optional parameter. When this property is not set, the messages which do not meet any of the conditions specified in the 'routes' section get routed to the built-in eventhub endpoint.",
           "$ref": "#/definitions/FallbackRouteProperties"
+        },
+        "enrichments": {
+          "description": "The list of user-provided enrichments that the IoT hub applies to messages to be delivered to built-in and custom endpoints. A maximum of 10 enrichments are allowed for paid hubs and a maximum of 2 enrichments are allowed for free hubs.",
```

Comment:
Would it be better to replace the quota numbers and replace it with a link to quota so that we don't have to change the swagger each time?

Reply:
Also, do we explicitly block fallback as an endpoint in enrichments?

Reply:
Do we need to mention Event Grid here or is that covered by built-in endpoints?

Reply:
Would prefer not mentioning Event Grid. Once we launch telemetry events on EG, we can think of updating this. For now, let's leave routing to EG out of enrichments.

Reply:
@cpjagan the quota numbers are mentioned elsewhere in the swagger as well, so I just followed that as practice. @ash2017, do we have a link to documentation that we may want to add here ?

@cpjagan, we do not block fallback as an endpoint in enrichments.

Reply:
Yes, we can replace details around 10 enrichments by stating:
For more details see [IoT Hub message enrichments]( aka.ms/telemetryoneventgrid ). 

This link will be updated when we have the final docs.